scripts/temp_convertElectrodes_step2.m

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Maybe we should call this file a template instead, since we will be saving this for each dataset

Reply:
added. Kept the same script name but added it in the first lines